_includes/deployments-script.html

```diff
@@ -12,6 +14,16 @@
       deploymentsData = JSON.parse(dataScript.textContent);
     }
 
+    // Load deployment hints
+    const hintsScript = document.getElementById('deployment-hints');
+    if (hintsScript) {
+      try {
+        deploymentHints = JSON.parse(hintsScript.textContent);
+      } catch (e) {
+        // console.warn('Could not parse deployment hints JSON', e);
+      }
+    }
```

Comment:
- Earlier versions used the JSON from the data module to set a global. That seems simpler?
- Hiding the error seems bad. If the data isn't there, something is seriously wrong, and it will be easier to debug if we're not hiding error messages.